pgjdbc/src/main/java/org/postgresql/jdbc/ArrayDecoding.java

```diff
@@ -0,0 +1,794 @@
+/*
+ * Copyright (c) 2018, PostgreSQL Global Development Group
```

Comment:
I guess this was originally written in 2018 ?

Reply:
Yep. Should I update these to 2020, since it is now getting merged in?

Reply:
please